doc/06-deploying-a-lambda.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deploying a Lambda
+---
+
+# Deploying a Lambda
+
+To deploy your project to AWS Lambda, we have provided a `Makefile` to make your life easier.
+Run `make` from the root of your project, and you will have a `build/function.zip` generated for you.
+
+Note that this uses [**Docker**](https://www.docker.com) underneath, so make sure you have it installed
+and set-up on your system.
+
+## Manual deploy
+
+To build the `function.zip` file manually, you need to run these commands:

```

Comment:
I noticed that that zip only contains one file called `bootstrap`. With other languages like node you mostly need to upload 1) your code AND 2) a `bootstrap` file which acts as the runtime (mostly in a layer).
Is that not necessary here? So does the Haskell lambda basically act as the runtime itself?

Reply:
Yep, that's it!

Reply:
Interesting, also found fuzzy-id's issue about. Would maybe be also worth a mention in the docs as to why that is better than the traditional approach.

Reply:
Hello, I am getting handler does not exist on the project when testing in AWS lambda, so there is one file called bootstrap, where is my handler function located?